CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 # UNRELEASED
 
+### feat: impersonating sender of requests to a local PocketIC instance
+
+`dfx canister call`, `dfx canister status`, and `dfx canister update-settings` take
```

Comment:
Please update the docs in https://github.com/dfinity/sdk/blob/master/docs/cli-reference/dfx-canister.mdx

Reply:
Done.